modular_skyrat/code/datums/mood_events/generic_negative_events.dm

```diff
@@ -3,3 +3,9 @@
 	description = "<span class='userdanger'>WHERE IS THE EPI PEN?!?!</span>\n"
 	mood_change = -10
 	timeout = 10 SECONDS
+
+//For when you say something really awful IC.
+/datum/mood_event/cringe
+	description = "<span class='boldwarning'>God, they're gonna put me in a cringe compilation after that...</span>\n"
```

Comment:
I feel like if it said something a bit more grounded like, "did I just said that out loud? God, that's embarrassing..." would go a long way, cause it saying "cringe comp" seems to be adding to the issue which is to stamp out the memes

Reply:
doing it